lib/membrane_framerate_converter.ex

```diff
@@ -0,0 +1,135 @@
+defmodule Membrane.FramerateConverter do
+  @moduledoc """
+  Element converts video to target constant frame rate, by dropping and duplicating frames as necessary
+  Input video may have constant or variable frame rate.
+  Element expects each frame to be received in separate buffer.
+  Additionally, presentation timestamps must be passed in each buffer's metadata.
+  """
+  use Membrane.Filter
+  use Bunch
+
+  alias Membrane.Buffer
+  alias Membrane.Caps.Video.Raw
+
+  require Membrane.Logger
+
+  def_options framerate: [
+                spec: Tuple,
```

Comment:
I would put `tuple()` as it is one of the basic types ([https://hexdocs.pm/elixir/1.12/typespecs.html](https://hexdocs.pm/elixir/1.12/typespecs.html))
```suggestion
                spec: tuple(),
```